tests/transform/test_genes_biodomains.py

```diff
@@ -0,0 +1,96 @@
+import pandas as pd
+
+from agoradatatools.etl.transform.genes_biodomains import count_grouped_total
+
+
+class TestCountGroupedTotal:
```

Comment:
Note: when count_grouped_total() gets moved to utils, this test should get moved too.